fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/ResourceDAOImpl.java

```diff
@@ -460,6 +460,19 @@ public void addResourceTypeCacheCandidate(String resourceType, Integer resourceT
         log.exiting(CLASSNAME, METHODNAME);
 
     }
+    
+    protected  Integer getResourceTypeIdFromCaches(String resourceType) {
+        Integer resourceTypeId;
+        // Get resourceTypeId from ResourceTypesCache first.
+        resourceTypeId = ResourceTypesCache.getResourceTypeId(resourceType);
+        // If no found, then get resourceTypeId from local newResourceTypeIds in case this id is already in newResourceTypeIds
```

Comment:
```suggestion
        // Get resourceTypeId from ResourceTypesCache first.
        Integer resourceTypeId= ResourceTypesCache.getResourceTypeId(resourceType);
        // If not found, then get resourceTypeId from local newResourceTypeIds in case this id is already in newResourceTypeIds
```
honestly, it should never hit newResourceTypeIds with the change in PR1010, but seems reasonable. 
small edits above